### DISCUSSION TITLE
Symfony stable versions suitable for production

### DISCUSSION THREAD

**Discussion by Bsd-us** [shinsenter/php#87] [Q&A] [answered]
Hi,

I'm building my app based on shinsenter/symfony but I couldn't find any stable versions suitable for production
I looked on https://hub.docker.com/r/shinsenter/php-archives
Am I missing something ? If not, maybe you could consider it ?

**Comment by shinsenter** [comment, marked as answer]
@Bsd-us 

Thank you for your question.

Currently, the [php-archives](https://hub.docker.com/r/shinsenter/php-archives) repository only supports reflecting the build versions of Docker images from the [main repository](https://hub.docker.com/r/shinsenter/php).

But you can pull the image from `shinsenter/symfony:latest`, tag it as `shinsenter/symfony:20240730-php8.3` using the below commands:

```shell
docker pull shinsenter/symfony:latest
docker tag  shinsenter/symfony:latest your-repo/symfony:20240730-php8.3
docker push your-repo/symfony:20240730-php8.3
```

Then use the image `your-repo/symfony:20240730-php8.3` as a base image to build containers for production.
